app/assets/javascripts/show-free-text-question.js

```diff
@@ -0,0 +1,31 @@
+window.GOVUKAdmin = window.GOVUKAdmin || {};
```

Comment:
I don't think we need these lines, since we've added a [file that sets these up](https://github.com/alphagov/content-performance-manager/blob/master/app/assets/javascripts/namespaces.js).